### PR TITLE
i18n: Corrected one p6s machanic CN translation.

### PR DESCRIPTION
**Pull request by richackard**
Corrected the Chinese translation of p6s `Synergy` mechanic.

### CODE REVIEW

File: ui/raidboss/data/06-ew/raid/p6s.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ const triggerSet: TriggerSet<Data> = {
           de: 'Geteilter Tankbuster',
           fr: 'Séparez les Tankbusters',
           ja: '2人同時タンク強攻撃',
-          cn: '坦克分摊死刑',
+          cn: '双T分散死刑',
```
Comment (MaikoTan) [inline ui/raidboss/data/06-ew/raid/p6s.ts:83]: I think the original one is enough for readability, is there any reason to change it to "雙T" instead?

Comment (ShadyWhite) [inline ui/raidboss/data/06-ew/raid/p6s.ts:83]:
```suggestion
          cn: '双T死刑',
```

Comment (ShadyWhite) [reply ui/raidboss/data/06-ew/raid/p6s.ts:83]: I think '双T死刑' is enough in this situation.

**Review by ShadyWhite — CHANGES_REQUESTED** [review]

File: ui/raidboss/data/06-ew/raid/p6s.ts (hunk 1 above)
Comment (MaikoTan) [reply ui/raidboss/data/06-ew/raid/p6s.ts:83]: Suggest: "分散死刑" by @Echoring as it is the only word that tanks should remember.

**Comment by MaikoTan** [comment]
@Akurosia Hey bro, I don't know deutsch but it looks like there are something wrong in the deutsche translation of this trigger either, which should announce tank to take the tank busters separately, but currently it looks like saying sharing?

**Pull request merged**